src/write_outputs/write_capacity.jl

```diff
@@ -26,6 +26,11 @@ function write_capacity(path::AbstractString, inputs::Dict, setup::Dict, EP::Mod
 		end
 	end
 
+	capacity_constraint_dual = zeros(size(inputs["RESOURCES"]))
+	for y in dfGen[dfGen.Max_Cap_MW.>0, :R_ID]
+		capacity_constraint_dual[y] = dual.(EP[:cMaxCap][y])
```

Comment:
Does `:cMaxCap` exist for all resources with `Max_Cap_MW > 0` or do they also need to have `New_Build` enabled?

Reply:
Ah yes it does, great.

```
  @constraint(EP, cMaxCap[y in intersect(dfGen[dfGen.Max_Cap_MW.>0, :R_ID], 1:G)], eTotalCap[y] <= dfGen[y, :Max_Cap_MW])
```